src/main/java/com/questhelper/steps/QuestStep.java

```diff
@@ -506,4 +513,91 @@ public BufferedImage getQuestImage()
 	{
 		return spriteManager.getSprite(SpriteID.TAB_QUESTS, 0);
 	}
+
+
+	public void showWorldTooltips(PanelComponent panelComponent, boolean showHovered)
+	{
+		if (showHovered)
+		{
+			showHoveredItem(panelComponent);
+		}
+		else
+		{
+			showMenuItem(panelComponent);
+		}
+	}
+
+	private void showHoveredItem(PanelComponent panelComponent)
```

Comment:
nit/naming: `renderHoveredItemTooltip` - and it shouldn't accept a `PanelComponent`